src/app/pages/homepage/homepage.html

```diff
@@ -6,55 +6,81 @@ <h1 class="mat-h1">Angular Material</h1>
       <h2> Material Design components for Angular</h2>
     </div>
     <div class="docs-header-start">
-     <a mat-raised-button class="docs-button" routerLink="/guide/getting-started">Get started</a>
+      <a mat-raised-button class="docs-button" routerLink="/guide/getting-started">Get started</a>
     </div>
   </div>
 </header>
 
 <main class="docs-homepage-promo">
-  <div class="docs-homepage-row">
-    <div class="docs-homepage-promo-img">
-      <docs-svg-viewer src="../assets/img/homepage/sprintzerotoapp.svg"
-                       [scaleToContainer]="true"></docs-svg-viewer>
+  <div class="docs-homepage-row docs-introduction">
+    <div class="docs-homepage-row-column">
+      <h2>High quality</h2>
+      <p>
+        Internationalized and accessible components for everyone. Well tested to ensure performance and reliability.
+      </p>
+      <p>Straightforward APIs with consistent cross platform behaviour.</p>
     </div>
```

Comment:
@jelbourn please feel free to add/suggest actually meaningful text here, I got most of this from the README of the components repo